flask_rest_orm/resources/utils.py

```diff
@@ -22,10 +48,34 @@ def query_from_request(model, request):
     :return: the items for the current page
     """
     builder = model.query
```

Comment:
A better name for this var would be `query`

Reply:
OK